pkg/resourcecollector/resourcecollector.go

```diff
@@ -343,6 +346,7 @@ func (r *ResourceCollector) GetResources(
 				}
 				for _, o := range objects {
 					runtimeObject, ok := o.(runtime.Unstructured)
+					logrus.Infof("line 349 runtime object: %+v", runtimeObject)
```

Comment:
nit: remove debug line

Reply:
My bad thought had removed it, cleaned it up.